.pre-commit-config.yaml

```diff
@@ -56,7 +56,7 @@ repos:
           - --remove-duplicate-keys
           - --remove-unused-variables
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 22.3.0
```

Comment:
This is creating problem to my installation:
```
$ pre-commit run -a                                                                                                                                                                                                                                               
[INFO] Initializing environment for https://github.com/oca/maintainer-tools.
[INFO] Initializing environment for https://github.com/myint/autoflake.
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-prettier.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-prettier:@prettier/plugin-xml@0.12.0,prettier@2.1.2.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-eslint.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-eslint:eslint@7.8.1.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/asottile/pyupgrade.
[INFO] Initializing environment for https://github.com/PyCQA/isort.
[INFO] Initializing environment for https://github.com/acsone/setuptools-odoo.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8:flake8-bugbear==20.1.4.
[INFO] Initializing environment for https://github.com/PyCQA/pylint.
[INFO] Initializing environment for https://github.com/PyCQA/pylint:pylint-odoo==5.0.5.
[INFO] Installing environment for https://github.com/oca/maintainer-tools.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/myint/autoflake.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/roberto/.cache/pre-commit/repo5ou3z5l4/py_env-python3/bin/python', '-mpip', 'install', '.')
return code: 1
expected return code: 0
stdout:
    Processing /home/roberto/.cache/pre-commit/repo5ou3z5l4
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
        Preparing wheel metadata: started
        Preparing wheel metadata: finished with status 'done'
    Collecting tomli>=1.1.0
      Using cached tomli-2.0.1-py3-none-any.whl (12 kB)
    Collecting mypy-extensions>=0.4.3
      Using cached mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
    Collecting typing-extensions>=3.10.0.0
      Using cached typing_extensions-4.1.1-py3-none-any.whl (26 kB)
    Collecting click>=8.0.0
      Using cached click-8.1.1-py3-none-any.whl (96 kB)
    Collecting pathspec>=0.9.0
      Using cached pathspec-0.9.0-py2.py3-none-any.whl (31 kB)
    Collecting platformdirs>=2
      Using cached platformdirs-2.5.1-py3-none-any.whl (14 kB)
    Building wheels for collected packages: black
      Building wheel for black (PEP 517): started
      Building wheel for black (PEP 517): finished with status 'error'
    Failed to build black
    
stderr:
      DEPRECATION: A future pip version will change local packages to be built in-place without first copying to a temporary directory. We recommend you use --use-feature=in-tree-build to test your packages with this new behavior before it becomes the default.
       pip 21.3 will remove support for this functionality. You can find discussion regarding this at https://github.com/pypa/pip/issues/7555.
      ERROR: Command errored out with exit status 1:
       command: /home/roberto/.cache/pre-commit/repo5ou3z5l4/py_env-python3/bin/python /home/roberto/.cache/pre-commit/repo5ou3z5l4/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py build_wheel /tmp/tmpvkrq9kg_
           cwd: /tmp/pip-req-build-22697kjv
      Complete output (174 lines):
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib
      copying src/_black_version.py -> build/lib
      creating build/lib/blib2to3
      copying src/blib2to3/pytree.py -> build/lib/blib2to3
      copying src/blib2to3/pygram.py -> build/lib/blib2to3
      copying src/blib2to3/__init__.py -> build/lib/blib2to3
      creating build/lib/blackd
      copying src/blackd/middlewares.py -> build/lib/blackd
      copying src/blackd/__main__.py -> build/lib/blackd
      copying src/blackd/__init__.py -> build/lib/blackd
      creating build/lib/black
      copying src/black/trans.py -> build/lib/black
      copying src/black/strings.py -> build/lib/black
      copying src/black/rusty.py -> build/lib/black
      copying src/black/report.py -> build/lib/black
      copying src/black/parsing.py -> build/lib/black
      copying src/black/output.py -> build/lib/black
      copying src/black/numerics.py -> build/lib/black
      copying src/black/nodes.py -> build/lib/black
      copying src/black/mode.py -> build/lib/black
      copying src/black/lines.py -> build/lib/black
      copying src/black/linegen.py -> build/lib/black
      copying src/black/handle_ipynb_magics.py -> build/lib/black
      copying src/black/files.py -> build/lib/black
      copying src/black/debug.py -> build/lib/black
      copying src/black/const.py -> build/lib/black
      copying src/black/concurrency.py -> build/lib/black
      copying src/black/comments.py -> build/lib/black
      copying src/black/cache.py -> build/lib/black
      copying src/black/brackets.py -> build/lib/black
      copying src/black/__main__.py -> build/lib/black
      copying src/black/__init__.py -> build/lib/black
      creating build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/tokenize.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/token.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/pgen.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/parse.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/literals.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/grammar.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/driver.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/conv.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/pgen2/__init__.py -> build/lib/blib2to3/pgen2
      copying src/blib2to3/PatternGrammar.txt -> build/lib/blib2to3
      copying src/blib2to3/Grammar.txt -> build/lib/blib2to3
      copying src/black/py.typed -> build/lib/black
      installing to build/bdist.linux-x86_64/wheel
      running install
      running install_lib
      creating build/bdist.linux-x86_64
      creating build/bdist.linux-x86_64/wheel
      creating build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/py.typed -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/__init__.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/__main__.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/brackets.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/cache.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/comments.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/concurrency.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/const.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/debug.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/files.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/handle_ipynb_magics.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/linegen.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/lines.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/mode.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/nodes.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/numerics.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/output.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/parsing.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/report.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/rusty.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/strings.py -> build/bdist.linux-x86_64/wheel/black
      copying build/lib/black/trans.py -> build/bdist.linux-x86_64/wheel/black
      creating build/bdist.linux-x86_64/wheel/blackd
      copying build/lib/blackd/__init__.py -> build/bdist.linux-x86_64/wheel/blackd
      copying build/lib/blackd/__main__.py -> build/bdist.linux-x86_64/wheel/blackd
      copying build/lib/blackd/middlewares.py -> build/bdist.linux-x86_64/wheel/blackd
      creating build/bdist.linux-x86_64/wheel/blib2to3
      copying build/lib/blib2to3/Grammar.txt -> build/bdist.linux-x86_64/wheel/blib2to3
      copying build/lib/blib2to3/PatternGrammar.txt -> build/bdist.linux-x86_64/wheel/blib2to3
      creating build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/__init__.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/conv.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/driver.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/grammar.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/literals.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/parse.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/pgen.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/token.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/pgen2/tokenize.py -> build/bdist.linux-x86_64/wheel/blib2to3/pgen2
      copying build/lib/blib2to3/__init__.py -> build/bdist.linux-x86_64/wheel/blib2to3
      copying build/lib/blib2to3/pygram.py -> build/bdist.linux-x86_64/wheel/blib2to3
      copying build/lib/blib2to3/pytree.py -> build/bdist.linux-x86_64/wheel/blib2to3
      copying build/lib/_black_version.py -> build/bdist.linux-x86_64/wheel
      running install_egg_info
      running egg_info
      creating src/black.egg-info
      writing src/black.egg-info/PKG-INFO
      writing dependency_links to src/black.egg-info/dependency_links.txt
      writing entry points to src/black.egg-info/entry_points.txt
      writing requirements to src/black.egg-info/requires.txt
      writing top-level names to src/black.egg-info/top_level.txt
      writing manifest file 'src/black.egg-info/SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      adding license file 'LICENSE'
      adding license file 'AUTHORS.md'
      writing manifest file 'src/black.egg-info/SOURCES.txt'
      Copying src/black.egg-info to build/bdist.linux-x86_64/wheel/black-0.1.dev1+gae2c075-py3.8.egg-info
      running install_scripts
      adding license file "LICENSE" (matched pattern "LICEN[CS]E*")
      adding license file "AUTHORS.md" (matched pattern "AUTHORS*")
      creating build/bdist.linux-x86_64/wheel/black-0.1.dev1+gae2c075.dist-info/WHEEL
      creating '/tmp/pip-wheel-ru3tg4je/tmpfdyabxwj/black-0.1.dev1+gae2c075-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
      adding '_black_version.py'
      adding 'black/__init__.py'
      adding 'black/__main__.py'
      adding 'black/brackets.py'
      adding 'black/cache.py'
      adding 'black/comments.py'
      adding 'black/concurrency.py'
      adding 'black/const.py'
      adding 'black/debug.py'
      adding 'black/files.py'
      adding 'black/handle_ipynb_magics.py'
      adding 'black/linegen.py'
      adding 'black/lines.py'
      adding 'black/mode.py'
      adding 'black/nodes.py'
      adding 'black/numerics.py'
      adding 'black/output.py'
      adding 'black/parsing.py'
      adding 'black-0.1.dev1+gae2c075.dist-info/RECORD'
      /tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools_scm/git.py:105: UserWarning: "/tmp/pip-req-build-22697kjv" is shallow and may cause errors
        warnings.warn(f'"{wd.path}" is shallow and may cause errors')
      Traceback (most recent call last):
        File "/home/roberto/.cache/pre-commit/repo5ou3z5l4/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 349, in <module>
          main()
        File "/home/roberto/.cache/pre-commit/repo5ou3z5l4/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 331, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/home/roberto/.cache/pre-commit/repo5ou3z5l4/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 248, in build_wheel
          return _build_backend().build_wheel(wheel_directory, config_settings,
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 244, in build_wheel
          return self._build_with_temp_dir(['bdist_wheel'], '.whl',
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 229, in _build_with_temp_dir
          self.run_setup()
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 174, in run_setup
          exec(compile(code, __file__, 'exec'), locals())
        File "setup.py", line 73, in <module>
          setup(
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 148, in setup
          return run_commands(dist)
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
          dist.run_commands()
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
          self.run_command(cmd)
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 1214, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
          cmd_obj.run()
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/wheel/bdist_wheel.py", line 361, in run
          wf.write_files(archive_root)
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/wheel/wheelfile.py", line 136, in write_files
          self.write(path, arcname)
        File "/tmp/pip-build-env-ap4ouf1j/overlay/lib/python3.8/site-packages/wheel/wheelfile.py", line 147, in write
          zinfo = ZipInfo(arcname or filename, date_time=get_zipinfo_datetime(st.st_mtime))
        File "/usr/lib64/python3.8/zipfile.py", line 362, in __init__
          raise ValueError('ZIP does not support timestamps before 1980')
      ValueError: ZIP does not support timestamps before 1980
      ----------------------------------------
      ERROR: Failed building wheel for black
    ERROR: Could not build wheels for black which use PEP 517 and cannot be installed directly
```
Instead if use patch below it make ``black`` to work again
```diff
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 77961333c..6104b592b 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -59,6 +59,7 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
+        additional_dependencies: ["click==8.0.4"]
   - repo: https://github.com/pre-commit/mirrors-prettier
     rev: v2.1.2
     hooks:
```